x-pack/plugins/ingest_manager/public/applications/ingest_manager/components/enrollment_instructions/manual/index.tsx

```diff
@@ -20,37 +20,79 @@ export const ManualInstructions: React.FunctionComponent<Props> = ({
   apiKey,
   kibanaCASha256,
 }) => {
-  const command = `
-./elastic-agent enroll ${kibanaUrl} ${apiKey.api_key}${
+  const enrollArgs = `${kibanaUrl} ${apiKey.api_key}${
     kibanaCASha256 ? ` --ca_sha256=${kibanaCASha256}` : ''
-  }
+  }`;
+  const macOsLinuxTarCommand = `
+./elastic-agent enroll ${enrollArgs}
 ./elastic-agent run`;
+
+  const linuxDebRpmCommand = `
+./elastic-agent enroll ${enrollArgs}
+systemctl enable elastic-agent
+systemctl start elastic-agent`;
+
+  const windowsCommand = `
+./elastic-agent enroll ${enrollArgs}
+./install-service-elastic-agent.ps1`;
+
   return (
     <>
       <EuiText>
         <FormattedMessage
           id="xpack.ingestManager.enrollmentInstructions.descriptionText"
-          defaultMessage="From the agent’s directory, run these commands to enroll and start the Elastic Agent. {enrollCommand} will write to your agent’s configuration file so that it has the correct settings. You can use this command to setup agents on more than one host."
+          defaultMessage="From the agent’s directory, run the appropriate commands to enroll and start an Elastic Agent. You can reuse these commands to setup agents on more than one machine."
+        />
+      </EuiText>
+      <EuiSpacer size="l" />
+      <EuiTitle size="xs">
+        <h4>
+          <FormattedMessage
+            id="xpack.ingestManager.enrollmentInstructions.windowsTitle"
+            defaultMessage="Windows"
+          />
+        </h4>
+      </EuiTitle>
+      <EuiSpacer size="s" />
+      <EuiCodeBlock fontSize="m" isCopyable={true} paddingSize="m">
+        <pre style={{ overflow: 'scroll' }}>{windowsCommand}</pre>
+      </EuiCodeBlock>
```

Comment:
```suggestion
      <EuiCodeBlock fontSize="m" whiteSpace="pre" isCopyable={true} paddingSize="m">
        {windowsCommand}
      </EuiCodeBlock>
```

Reply:
@jen-huang I tried the whiteSpace pr and the result is not great the copy button is over the command to copy

Reply:
@nchaulet good point, then let's add a `styled` component instead instead of having inline styles on these `<pre>` elements?

Reply:
Sure 👍

Reply:
I also opened an EUI issue to see if they can fix this behavior in a generic way: https://github.com/elastic/eui/issues/3815